media/css/protocol/components/_download-button.scss

```diff
@@ -0,0 +1,234 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../../../node_modules/@mozilla-protocol/core/protocol/css/includes/lib";
```

Comment:
Maybe I'm crazy, but after a little local experimenting I'm not sure we actually need to re-import this in every file. It's become such a habit for so long I haven't even questioned it before, but now I wonder if it's a legacy from Sandstone/Less that isn't needed in Sass. It needs more thorough testing. But I don't like the length and fragility of this relative path so if we don't have to repeat it everywhere that would be nice. Not relevant to this PR but something to think about.

Reply:
Godd point - let's investigate further in Protocol itself 👍